packages/web-components/fast-foundation/src/tooltip/tooltip.ts

```diff
@@ -131,6 +131,14 @@ export class Tooltip extends FoundationElement {
                     this.handleAnchorMouseOut,
                     { passive: true }
                 );
+                this.anchorElement.addEventListener("focusin", this.handleAnchorFocusIn, {
```

Comment:
Do you need to tear down old listeners like on (line 119-120)?

Reply:
Ah, yes. Added.